client/src/components/graph/graph.tsx

```diff
@@ -126,6 +126,8 @@ class Graph extends React.Component<{}, GraphState> {
     return !shallowEqual(props.watchProps, prevProps.watchProps);
   }
 
+  private graphRef = React.createRef<HTMLDivElement>();
```

Comment:
This is used to prevent scrolling on the graph component at mount time.

Reply:
this is also a great context to have in the code 🙏